docs/pages/versions/unversioned/index.md

```diff
@@ -31,7 +33,9 @@ The easiest way to create a bare React Native app with support for the Expo SDK
 '$ npx create-react-native-app my-app',
 ]} cmdCopy="npx create-react-native-app my-app" />
 
-You can install and import any Expo SDK package using the `npx expo install` command. If you have an existing app that you would like to add Expo SDK packages to, read about [integrating Expo SDK into existing apps](/bare/existing-apps).
+<BoxLink title="Existing apps" href="/bare/installing-expo-modules" description={<>Projects that were created with <InlineCode>npx react-native init</InlineCode> require additional setup to use the Expo SDK.</>} />
+
+<BoxLink title="Using libraries" description="Learn how to install Expo SDK packages in your project." href="/workflow/using-libraries" />
```

Comment:
boxlink looks a bit weird under code blocks because it's slightly more indented:

<img width="1175" alt="image" src="https://user-images.githubusercontent.com/90494/188288489-5f79f5f4-8b39-4e2f-9813-165ad51ad61e.png">

not a deal breaker for this change, just noting it. no change needed in this pr
but

Reply:
@brentvatne It's actually not, that's your eyes playing tricks because of the light and dark 🤓 
![image](https://user-images.githubusercontent.com/852069/188290768-b5d8a36f-cf7d-4f30-a210-12846ddb2735.png)